github-webooks/iam-roles.tf

```diff
@@ -0,0 +1,168 @@
+# The lambda receive role + policy allows:
+# * Writing to CloudWatch
+# * Writing to the firehose
+# * Writing to SNS
+resource "aws_iam_role" "lambda_github_webhooks_receive" {
+    name = "lambda-github-webhooks-receive"
+    assume_role_policy = <<EOF
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Effect": "Allow",
+      "Principal": {
+        "Service": "lambda.amazonaws.com"
+      },
+      "Action": "sts:AssumeRole"
+    }
+  ]
+}
+EOF
+}
+
+resource "aws_iam_role_policy" "lambda_github_webhooks_receive_policy" {
+  name = "lambda-github-webhooks-receive"
```

Comment:
Nit: we've been using 4-space indentation everywhere; I don't hugely mind 2 or 4, but both in the same file irks me.